.github/workflows/ci-master.yml

```diff
@@ -9,20 +9,15 @@ jobs:
   install-and-build:
     runs-on: ubuntu-latest
 
-    timeout-minutes: 30
-
-    strategy:
-      matrix:
-        node-version: [18.x, 20.x]
```

Comment:
not sure why were were building and linting on multiple versions of node. we only need to run the tests on multiple versions.